app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
+    post = Post.find(params[:post_id])
+    comments = post.comments
+    render json: comments
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
```

Comment:
- [ ] I am unable to create a comment using the API on Postman. You are expecting a `comment_params` method but you did not create this method. Kindly create the method towards the end of the comments controller.

  ```
  ...
  private

  def comment_params
    params.require(:comment).permit(:text)
  end
  ```

  ![idodw](https://user-images.githubusercontent.com/36542986/231746939-f4dc3ae5-9468-4677-86e8-0a62193afeca.jpg)